api/versioninfo.php

```diff
@@ -0,0 +1,117 @@
+<?php
+  /**
+    Finds the latest versions of downloads and returns a JSON blob with the data. The first *.download_info file in each folder will be checked.
```

Comment:
At present, visiting this page via IIS spews quite a few errors that block the JSON from being correctly output.

<br />
<b>Warning</b>:  scandir(../android,../android): The system cannot find the file specified. (code: 2) in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>70</b><br />
<br />
<b>Warning</b>:  scandir(../android): failed to open dir: No such file or directory in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>70</b><br />
<br />
<b>Warning</b>:  scandir(): (errno 2): No such file or directory in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>70</b><br />
<br />
<b>Warning</b>:  array_filter() expects parameter 1 to be array, boolean given in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>71</b><br />
<br />
<b>Warning</b>:  usort() expects parameter 1 to be array, null given in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>72</b><br />
<br />
<b>Warning</b>:  Invalid argument supplied for foreach() in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>76</b><br />
<br />
<b>Warning</b>:  scandir(../ios,../ios): The system cannot find the file specified. (code: 2) in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>70</b><br />
<br />
<b>Warning</b>:  scandir(../ios): failed to open dir: No such file or directory in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>70</b><br />
<br />
<b>Warning</b>:  scandir(): (errno 2): No such file or directory in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>70</b><br />
<br />
<b>Warning</b>:  array_filter() expects parameter 1 to be array, boolean given in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>71</b><br />
<br />
<b>Warning</b>:  usort() expects parameter 1 to be array, null given in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>72</b><br />
<br />
<b>Warning</b>:  Invalid argument supplied for foreach() in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>76</b><br />
<br />
<b>Warning</b>:  scandir(../mac,../mac): The system cannot find the file specified. (code: 2) in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>70</b><br />
<br />
<b>Warning</b>:  scandir(../mac): failed to open dir: No such file or directory in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>70</b><br />
<br />
<b>Warning</b>:  scandir(): (errno 2): No such file or directory in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>70</b><br />
<br />
<b>Warning</b>:  array_filter() expects parameter 1 to be array, boolean given in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>71</b><br />
<br />
<b>Warning</b>:  usort() expects parameter 1 to be array, null given in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>72</b><br />
<br />
<b>Warning</b>:  Invalid argument supplied for foreach() in <b>C:\Keyman\websites\downloads.keyman.com\api\versioninfo.php</b> on line <b>76</b><br />